### PR TITLE
Sscs 10327

### PR DESCRIPTION
**Pull request by vinothon84**
Pull Request for adding hearing Date to excludeDate.

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/postponementrequest/PostponementRequestAboutToStartHandlerTest.java [hunk 1]
```diff
@@ -79,6 +81,19 @@ static Hearing getHearing(int hearingId) {
                 .build()).build();
     }
 
+    public static HearingOptions getHearingOptions() {
+        HearingOptions hearingOptions = HearingOptions.builder()
+                .wantsToAttend(YES)
+                .languageInterpreter(YES)
+                .languages("Spanish")
+                .signLanguageType("A sign language")
+                .arrangements(Arrays.asList("hearingLoop", "signLanguageInterpreter"))
```
Comment (nickwest1) [inline src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/postponementrequest/PostponementRequestAboutToStartHandlerTest.java:90]: Arrays.asList should be a constant

**Review by nickwest1 — CHANGES_REQUESTED** [review]
Only the one change that's worth applying

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/postponementrequest/PostponementRequestAboutToStartHandlerTest.java [hunk 2]
```diff
@@ -79,6 +81,19 @@ static Hearing getHearing(int hearingId) {
...
+                .other("Yes, this...")
+                .build();
+
+        return  hearingOptions;
```
Comment (MohammedHMCTS) [inline src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/postponementrequest/PostponementRequestAboutToStartHandlerTest.java:94]: Just add the return tot he actual builder no point in this if nothing else is happening.

**Review by MohammedHMCTS — APPROVED** [review]

**Review by nickwest1 — APPROVED** [review]

**Comment by vinothon84** [comment]
Had a chat with BA team and I need to my code changes to ActionPostponementRequest rather than PostponementRequest so will create a new branch to move my code changes.

**Pull request closed without merging**